scripts/src/Utils.hs

```diff
@@ -1,14 +1,14 @@
 module Utils where
 
-import Control.Exception
-import Control.Monad.IO.Class (MonadIO, liftIO)
-import Data.Maybe
-import Distribution.Simple.Utils
-import Distribution.Verbosity
-import System.Directory
-import System.Environment
-import System.FilePath
-import System.IO.Error
+import Prologue
+
+import Distribution.Simple.Utils (copyDirectoryRecursive, shortRelativePath)
+import Distribution.Verbosity    (silent)
+import System.Directory          (copyFile, createDirectoryIfMissing,

```

Comment:
Just a note, like all lists, we wrap lists with a leading comma rather than a trailing comma.